libraries/provision/ansible/playbooks/start-sg-accel.yml

```diff
@@ -24,7 +25,8 @@
 
   vars:
     sync_gateway_config_filepath:
-    couchbase_server_primary_node: "{{ hostvars[groups.couchbase_servers[0]].ansible_host }}"
+    # couchbase_server_primary_node: "{{ hostvars[groups.couchbase_servers[0]].ansible_host }}"
```

Comment:
Can these comments be removed?

Reply:
I had it for reference, anyways removed the comments as discussed